trojsten/contests/templates/trojsten/contests/parts/progress.html

```diff
@@ -7,7 +7,14 @@
 <div class="roundprogressbar" id="progressbar-{{ round.id }}" data-id="{{ round.id }}" data-precision="{{ remaining|progress_time_precision }}" data-isresults={{ results }}>
   <div class="row">
     {% if round.second_phase_running %}
-      <div class="col-md-5"><em>{% trans "Submit programs until" %}: {{ end }}</em></div>
+      <div class="col-md-5"><em>
+        {% if submit_programs %}
+          {% trans "Submit programs until" %}:
+        {% else %}
+          {% trans "Solve with hints until" %}:
+        {% endif %}
+        {{ end }}</em>
+      </div>
```

Comment:
Nie som uplne nadseny z tohto ifu (aj tam nizsie). Asi by bolo lepsie si poslat rovno tie hlasky.

Ale asi to prezijem kym su tie hlasky len 2.